src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -155,8 +155,7 @@ export const DetailsForm: React.FC<DetailsFormProps> = ({
   const mapIconUrl = useMemo(() => {
     const file = values.mapIconImageFile;
     if (file && file.length > 0) return URL.createObjectURL(file[0]);
-    return values.mapIconImageUrl;
-  }, [values.mapIconImageFile, values.mapIconImageUrl]);
+  }, [values.mapIconImageFile]);
```

Comment:
I expect we are probably going to want to be assessing this whole piece of code, and potentially removing it entirely; not just removing small bits of it.

Reply:
And all of the related bits of code/usages that will then flow on from that.